test/jdk/javax/swing/JFileChooser/AquaDefaultButtonTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.JButton;
+import javax.swing.JFileChooser;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+
+/*
+ * @test
+ * @bug 8300084
+ * @key headful
+ * @summary Test to check if getDefaultButton() return valid default approve button.
+ * @requires (os.family == "mac")
+ * @run main AquaDefaultButtonTest
+ */
+
+public class AquaDefaultButtonTest {
+
+    public static void main(String[] args) throws Exception {
+        UIManager.setLookAndFeel("com.apple.laf.AquaLookAndFeel");
+        SwingUtilities.invokeAndWait(new Runnable() {
+            public void run() {
+                JFileChooser fileChooser = new JFileChooser();
+                JButton defaultApproveButton = fileChooser.getUI().getDefaultButton(fileChooser);
+                if (defaultApproveButton == null) {
+                    throw new RuntimeException("getDefaultButton() method returns null for Aqua L&F!");
+                }
```

Comment:
If you choose shorter names for variables, the line would fit into 80-column limit. Now the two lines are nearly 100 characters long, though not a big issue.

Reply:
Could you please the copyright year in `AquaFileChooserUI.java`?

Reply:
Updated.